portality/crosswalks/article_form.py

```diff
@@ -43,7 +43,7 @@ def form2obj(cls, form, add_journal_info=True):
 
         # keywords
         keywords = form.keywords.data
-        if keywords is not None and keywords is not []:
+        if keywords is not None:
```

Comment:
```suggestion
        if keywords is not None and len(keywords) > 0:
```

It's true that the expression didn't have the required effect, but it's better to replace with a working check than remove it entirely.